Source/Dafny/Resolver.cs

```diff
@@ -2334,7 +2334,7 @@ public class ModuleBindings {
       var receiver = f.IsStatic ? (Expression)new StaticReceiverExpr(tok, cl, true) : new ImplicitThisExpr(tok);
       var fn = new FunctionCallExpr(tok, f.Name, receiver, tok, tok, f.Formals.ConvertAll(Expression.CreateIdentExpr));
       var post = new AttributedExpression(new BinaryExpr(tok, BinaryExpr.Opcode.Eq, r, fn));
-      var method = new Method(tok, f.Name, f.HasStaticKeyword, false, f.TypeArgs,
+      var method = new Method(f.tok, f.Name, f.HasStaticKeyword, false, f.TypeArgs,
```

Comment:
```suggestion
      var method = new Method(tok, f.Name + "_by_method", f.HasStaticKeyword, false, f.TypeArgs,
```

I understand why you are doing that: That way, this method has the same symbol position as the function itself.
However, can you elaborate on why this results in a better user experience?

In my short-sighted mind, I pretty much see the by-method as syntactic sugar so 
that I don't need to write a separate method. So, I wouldn't mind that it results in two different verification symbols, even if the by-method symbol points to the "method" keyword - but it should be displayed with f.name + "'s method"

Reply:
Interesting idea, but your proposed change doesn't achieve that. No names are passed to the IDE by the symbolStatus API, only ranges, so we should only return ranges that map to names in the IDE.

Reply:
Yes, my change does not achieve that, I was just suggesting a starting point.

Wouldn't it be the right moment to now evolve your API as I suggested in the past, to have a separate "DisplayName" from the "Range" in your API, and not assume that the range points to a valid name that you have to painfully and manually extract in the IDE?

Reply:
I prefer the current UI. We could discuss this design but I suggest we first fix the bug that the current UI has.